project_5.html

```diff
@@ -0,0 +1,697 @@
+							<a class="pj-part"
+								data-bs-content="<img src='./assets/images/project_5/ops_project_5_mini_mp3.webp' height='150px' width='auto'>"
+								data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0">
+								DFPlayer Mini Module
+							</a>
+						</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>Mini USB Cable</td>
+						<td>1</td>
+					</tr>
+				</tbody>
+			</table>
+			<!-- End Part Table -->
+
+
+
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-schematics">Schematics</h2>
+			<div class="accordion mb-5" id="schemAccordian">
+				<div class="accordion-item">
+					<h2 class="accordion-header">
+						<button class="accordion-button" type="button" data-bs-toggle="collapse"
+							data-bs-target="#schemCollapse1" aria-expanded="true" aria-controls="schemCollapse1">
+							Schematic A (External Pull-Up Resistor Circuit)
+						</button>
+					</h2>
+					<!--Pin to page?-->
+					<div id="schemCollapse1" class="accordion-collapse collapse show"
+						data-bs-parent="#schemAccordian">
+						<div class="accordion-body text-center">
+							<img src='assets/images/project_5/ops23_project5_schemA.png' class="pj-schem">
+						</div>
+					</div>
+				</div>
+				<div class="accordion mb-5" id="schemAccordian">
+				<div class="accordion-item">
+					<h2 class="accordion-header">
+						<button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
+							data-bs-target="#schemCollapse2" aria-expanded="true" aria-controls="schemCollapse2">
+							Schematic B (Internal Pull-Up Resistor Circuit)
+						</button>
+					</h2>
+					<!--Pin to page?-->
+					<div id="schemCollapse2" class="accordion-collapse collapse" data-bs-parent="#schemAccordian">
+						<div class="accordion-body text-center">
+							<img src='assets/images/project_5/ops23_project5_schemB.png' class="pj-schem"
+								style="max-width: 500px;">
+						</div>
+					</div>
+				</div>
+				<div class="accordion-item">
+					<h2 class="accordion-header">
+						<button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
+							data-bs-target="#schemCollapse3" aria-expanded="true" aria-controls="schemCollapse3">
+							Schematic C (iPoduino v2)
+						</button>
+					</h2>
+					<!--Pin to page?-->
+					<div id="schemCollapse3" class="accordion-collapse collapse" data-bs-parent="#schemAccordian">
+						<div class="accordion-body text-center">
+							<img src='assets/images/project_5/ops_23_project5_schemC.png' class="pj-schem"
+								style="max-width: 500px;">
+						</div>
+					</div>
+				</div>
+			</div>
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-instructions">Instructions</h2>
+			<h3 class="mb-3" id="pj-heading-checkpoint1">Checkpoint 1</h3>
+			<ol class="mb-5 list-group">
+				<li class="list-group-item">
+					1. Connect the Arduino Nano to your computer and open the Arduino IDE. You will write a small program that echos small byte-sized values between the Nano and your computer.
```

Comment:
You shouldn't need to add numbering to each list item manually. Instead, add the "list-group-numbered" class to the parent <ol> tag. Please apply this to all other instances on this page.

Reply:
![image](https://github.com/uci-ieee/ops-webpages-2023-2024/assets/140114907/f52e3aed-e23e-4b63-9d7c-b373db99da68)
currently getting this error doing so.

Reply:
I fixed this error and it's now working :)